app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,26 @@
+class AccountActivationsController < ApplicationController
+  before_action :find_user, only: %i(edit)
+
+  def edit
+    if !@user.activated? && @user.authenticated?(:activation, params[:id])
+      @user.activate
+      log_in @user
+      flash[:success] = t(".success")
+      redirect_to @user
+    else
+      flash[:danger] = t(".danger")
+      redirect_to root_url
+    end
+    binding.pry
+  end
+
+  private
+
+  def find_user
+    @user = User.find_by(email: params[:email])
```

Comment:
bỏ () ko cần thiết đi em
```suggestion
    @user = User.find_by email: params[:email]
```